fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/Literal.java

```diff
@@ -85,12 +67,12 @@ public DataType getDataType() throws UnboundException {
 
     @Override
     public String toSql() {
-        return value.toString();
+        return String.valueOf(getValue());
```

Comment:
NullLiteral and BooleanLiteral should return upper case in toSql and toString

Reply:
done